lib/xcode.js

```diff
@@ -267,6 +262,6 @@ function clearInternalCache () {
   });
 }
 
-export { getPath, getVersion, getAutomationTraceTemplatePath, getMaxIOSSDK,
+export default { getPath, getVersion, getAutomationTraceTemplatePath, getMaxIOSSDK,
```

Comment:
why are we changing this to export default?

and if we want to, then we need to take the spaces out of the object, so it looks like `export default {getPath, ...}`